From 5da6e2e3708cb0b655894dd58030ecc04ef3bd97 Mon Sep 17 00:00:00 2001
From: Li Xinwei <1326710505@qq.com>
Date: Fri, 27 Nov 2020 18:19:37 +0800
Subject: [PATCH 14/17] Fix qdbusviewer.exe missing version and copyright
 information

The rc file of qdbusviewer.exe should be generated by the build system.

Change-Id: I038d5a092d95089b5f7ff36e48c2657dea230772
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
(cherry picked from commit 3385dea6e3b69c76fa3a1ec176eee3d84d20f7b8)
---
 src/qdbus/qdbusviewer/qdbusviewer.pro | 2 +-
 src/qdbus/qdbusviewer/qdbusviewer.rc  | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)
 delete mode 100644 src/qdbus/qdbusviewer/qdbusviewer.rc

diff --git a/src/qdbus/qdbusviewer/qdbusviewer.pro b/src/qdbus/qdbusviewer/qdbusviewer.pro
index 02ec7a2d..07a47cd4 100644
--- a/src/qdbus/qdbusviewer/qdbusviewer.pro
+++ b/src/qdbus/qdbusviewer/qdbusviewer.pro
@@ -23,7 +23,7 @@ mac {
 }
 
 win32 {
-    RC_FILE = qdbusviewer.rc
+    RC_ICONS = images/qdbusviewer.ico
 }
 
 load(qt_app)
diff --git a/src/qdbus/qdbusviewer/qdbusviewer.rc b/src/qdbus/qdbusviewer/qdbusviewer.rc
deleted file mode 100644
index c4b1d60b..00000000
--- a/src/qdbus/qdbusviewer/qdbusviewer.rc
+++ /dev/null
@@ -1 +0,0 @@
-IDI_ICON1		ICON		DISCARDABLE	"images/qdbusviewer.ico"
-- 
2.31.1

